redhat-appstudio/build-service
# Add new component image controller

#### What does this MR do?

This MR introduces a new controller that watches all Component related PipelineRuns and if any finish successfully, then checks for a newly built component image and updates corresponding component configuration.

#### How to test this MR?

1. Create an Application
2. Create a Component of the Application
3. Wait until initial build succeeds
4. Trigger a new build via the Component webhook
5. Wait until the webhook build finishes
6. Check the Component, it should contain the new image under `Spec.Build.ContainerImage`
